src/plugin/timeline.js

```diff
@@ -16,7 +16,9 @@
  * performance for large files
  * @property {string} fontFamily='Arial'
  * @property {number} fontSize=10 Font size of labels in pixels
- * @property {function} formatTimeCallback=→00:00
+ * @property {number} startSeconds=0 Start time in second for begining timeline rule
+ * @property {string} formatTime='MM:SS' default time format for timeline rule, can be change to'HH:MM:SS'
+ * @property {function} formatTimeCallback=→00:00 (or 00:00:00 depend of formatTime property)
```

Comment:
`depending on` instead of `depend of`

and can you remove the arrow: `→`

Reply:
OK, Arrow exist in original file, it's not me this time ;-)